### PR TITLE
Feat : Added Toast 

### PR DESCRIPTION
**Pull request by srejitk**
This Merge Request adds the following :

1. Toast Component and Variants
2. Documentation for the above

Toast

### CODE REVIEW

**Comment by nishasen** [comment]
I like your designs, its clean and good use of colours.

File: docs/Toasts.html [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Toasts</title>
+    <link rel="stylesheet" href="/main.css">
+    <link rel="stylesheet" href="/stylesheets/components/Toasts/Toast.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/night-owl.min.css"
+        integrity="sha512-i5X6Fdn/ZqvGSqPrdMa3FgcpXM/Nr6YccSKFYT93zljl/HZDEpvBbE5Pxp91eiWGccZLrL/LDQJd7fjTRYsVaA=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
```
Comment (nishasen) [inline docs/Toasts.html:16]: Add links in separate css file

File: docs/Toasts.html [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+</head>
+
+<body>
+    <div class="main-container">
+        <!-- Header HTML -->
+        <header id="header" class="iris__header">
+            <nav class="iris__navbar">
+                <div class="iris__navbar__logo__section">
+                    <a class="home__section">
```
Comment (nishasen) [inline docs/Toasts.html:25]: add href attribute here

**Review by nishasen — APPROVED** [review]
Good and clean code, I like the comments you have added. Its easy to understand the code using that

File: docs/Toasts.html (hunk 1 above)
Comment (srejitk) [reply docs/Toasts.html:16]: Yes, will do. Thank You.

File: docs/Toasts.html (hunk 2 above)
Comment (srejitk) [reply docs/Toasts.html:25]: Yes, added in the latest merge.

**Pull request merged**